src/packaging/webots_distro.c

```diff
@@ -942,7 +943,9 @@ static void create_file(const char *name, int m) {
       fprintf(fd, "' | osascript\n");
       fprintf(fd, "chmod -Rf go-w \"/Volumes/%s\"\n", application_name);
       fprintf(fd, "sync\n");
+      fprintf(fd, "sleep 5\n");
       fprintf(fd, "sync\n");
+      fprintf(fd, "sleep 5\n");
```

Comment:
Adding sleeps here fixes the creation of the package on travis for macOS.

I am currently testing the package creation on my fork before merging this: https://travis-ci.org/DavidMansolino/webots-test/builds/547575865

Reply:
Sounds like sorcery... Thank you Apple! 🙈

Reply:
It seems to work fine: https://github.com/DavidMansolino/webots-test/releases/tag/nightly_19_6_2019